docs/configuration.rst

```diff
@@ -121,12 +121,37 @@ policy.
     configured.
 
     .. note::
-       The "special" source values of ``'self'``, ``'unsafe-inline'``, ``'unsafe-eval'``,
-       ``'strict-dynamic'``, ``'none'``, etc. must be quoted!  e.g.: ``"default-src": ["'self'"]``.
-       Without quotes they will not work as intended.
+
+       The CSP keyword values of ``'self'``, ``'unsafe-inline'``, ``'strict-dynamic'``, etc. must be
+       quoted!  e.g.: ``"default-src": ["'self'"]``.  Without quotes they will not work as intended.
+       
+       New in version 4.0 are CSP keyword constants. Use these to minimize quoting mistakes and typos.
+       
+       The following CSP keywords are available:
+
+           * ``NONE`` = ``"'none'"``
+           * ``REPORT_SAMPLE`` = ``"'report-sample'"``
+           * ``SELF`` = ``"'self'"``
+           * ``STRICT_DYNAMIC`` = ``"'strict-dynamic'"``
+           * ``UNSAFE_ALLOW_REDIRECTS`` = ``"'unsafe-allow-redirects'"``
+           * ``UNSAFE_EVAL`` = ``"'unsafe-eval'"``
+           * ``UNSAFE_HASHES`` = ``"'unsafe-hashes'"``
+           * ``UNSAFE_INLINE`` = ``"'unsafe-inline'"``
+           * ``WASM_UNSAFE_EVAL`` = ``"'wasm-unsafe-eval'"``
+
+       Example usage:
+       
+       .. code-block:: python
        
-       Consider using the ``csp.constants`` module to get these values to help avoiding quoting
-       errors or typos, e.g., ``from csp.constants import SELF, STRICT_DYNAMIC``.
+            from csp.constants import SELF, STRICT_DYNAMIC
+
+            CONTENT_SECURITY_POLICY = {
+                "DIRECTIVES": {
+                    "default-src": [SELF, "cdn.example.net"],
+                    "script-src": [SELF, STRICT_DYNAMIC],
+                    "style-src": [SELF],
+                }
+            }
```

Comment:
These changes were meant for the last commit adding constants but got added to the wrong commit locally. Oops.